src/partition/repart.c

```diff
@@ -4904,7 +4908,7 @@ static int run(int argc, char *argv[]) {
         if (!context)
                 return log_oom();
 
-        r = context_read_definitions(context, arg_definitions, arg_root);
+        r = context_read_definitions(context, (const char * const*)arg_definitions, arg_root);
```

Comment:
I'd probably just drop the 'const' from the context_read_definitions() parameter. constant arrays of strings are a mess, you have to cast all the time. Hence so far we said: don't bother with const if you deal with a `char**` `strv` array...